NMDSdevopsServiceAdm/DataEngineering
# Validation check related cols

# Description
Add a custom check to the validation checks available
Create custom rule for care and primary service type
Add rule to validation jobs
Create unit tests

# How to test
Unit tests passing
Run in branch: https://eu-west-2.console.aws.amazon.com/states/home?region=eu-west-2#/v2/executions/details/arn:aws:states:eu-west-2:344210435447:execution:validation-check-related-cols-Ind-CQC-Filled-Post-Estimates-Pipeline:f6fbbb81-62f1-4cc3-92d8-f9c8e299deb9

# Developer checklist
- [x] Linked to Trello ticket: https://trello.com/c/EClhpkGQ/780-pydeequ-add-validation-for-care-home-and-primary-service-type-match
- [x] Documentation up to date: https://skillsforcare.atlassian.net/wiki/spaces/DE/pages/1115095041/Data+Quality+Validation
